base/base.go

```diff
@@ -0,0 +1,146 @@
+package base
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"time"
+
+	jose "gopkg.in/square/go-jose.v2"
+	"gopkg.in/square/go-jose.v2/jwt"
+)
+
+const NotifyBaseURL = "https://rest-api.notify.gov.au"
+
+type Response struct {
+	response *http.Response
+	body     *bytes.Buffer
+	Error    error
+}
+
+func (resp Response) JSON(v interface{}, at ...string) Response {
```

Comment:
@dalehurwitz see platforms PR (will DM link). I'd actually like to unexport this base package by putting it inside an `internal` folder per the comment in the PR link above. If we decide later that we want to export it, it's easy to do.